app/styles/_base.scss

```diff
@@ -109,6 +109,21 @@ input {
   font-weight: inherit;
 }
 
+button.link {
+  border: 0;
+  color: $link-color-default;
+  font-size: 14px;
+  height: 20px;
+  justify-content: left;
+  letter-spacing: 0;
+  width: 40%;
+}
+
+button.link:hover {
+  background: transparent url('/images/recovery_code_replace.svg') center left no-repeat;
```

Comment:
I think you can move the `background` style [here](https://github.com/mozilla/fxa-content-server/blob/0fd2fec9e3fe27c0196b608a411b14ee1375436c/app/styles/modules/_settings-totp.scss#L115).